Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.3.0"
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+FastTransforms = "057dd010-8810-581a-b7be-e3fc3b93f78c"
```

Comment:
What is it needed for?

Reply:
Did you compare with https://github.com/JuliaMath/NFFT.jl?

Reply:
Not really.